rest-api/src/main/resources/definitions/paged_resources/forward_cursor_paged_resource_list.yml

```diff
@@ -0,0 +1,21 @@
+type: object
+discriminator: items
+description: |
+    One page of records from a source that only allows you to fetch the next page of 
+    resources (no random access via an index to anywhere in the set of records). The 
+    key to retrieve the next page of records is provided by the `offsetKey` value.
+properties:
```

Comment:
We won't because we bump the version number. The JSON will still work for the older versions of the SDK.